meta-lmp-base/recipes-support/resize-helper/resize-helper_0.1.bb

```diff
@@ -5,7 +5,7 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-2-Clause;md5=cb641bc04cda31
 
 inherit allarch systemd
 
-RDEPENDS:${PN} += "e2fsprogs-resize2fs gptfdisk parted util-linux-findmnt"
```

Comment:
Please rdepend on util-linux-fdisk and util-linux-blockdev, which are now required.